src/components/Lightboxed/SearchResults.vue

```diff
@@ -111,8 +111,7 @@ export default {
 
   methods: {
     search (query) {
-      this.results = []
-      this.messagesLoad({ query })
+      this.messagesLoad({ query }, true)
```

Comment:
@Fajfa we might have a little problem here (my fault) -- search will aggregate all messages overall searches -- this will use up too much memory; can we somehow reset the state, after new messages get pulled? A flag or something